catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/QueryResponseImpl.java

```diff
@@ -302,7 +302,7 @@ private Result handleTake() {
         Result result = null;
         try {
             result = queue.take();
-            if (result == poisonPillResult) {
+            if (result == POISON_PILL_RESULT) {
```

Comment:
Wow the poisonPillResult has been here awhile. I'm assuming this was a long oversight. So I went ahead and made the change. @pklinef or @stustison do you know if we actually intended it the way it was.